### PR TITLE
[IMP] industry_restaurant: add website homepage

### PR DESCRIPTION
**Pull request by dhrs-odoo**
[IMP] industry_restaurant: add website homepage

### CODE REVIEW

File: industry_restaurant/static/src/binary/ir_attachment/1375.jpg
Comment (vava-odoo) [inline industry_restaurant/static/src/binary/ir_attachment/1375.jpg:1]: 1374 and 1375 are the same. Can't we keep only one?

File: industry_restaurant/static/src/binary/ir_attachment/1379-food.jpg
Comment (vava-odoo) [inline industry_restaurant/static/src/binary/ir_attachment/1379-food.jpg:1]: Same comment for 1378 and 1379

File: industry_restaurant/demo/website_view.xml [hunk 1]
```diff
@@ -1,5 +1,82 @@
...
+        <field name="type">qweb</field>
+        <field name="website_id" ref="website.default_website"/>
+        <field name="arch" type="xml">
+            <t name="Homepage" t-name="website.homepage">
+                <t t-call="website.layout">
+                    <t t-set="pageName" t-value="'homepage'"/>
+                    <div id="wrap" class="oe_structure oe_empty">
+                        <section class="s_cover parallax s_parallax_is_fixed bg-black-50 o_colored_level pb256 pt256" data-scroll-background-ratio="1" data-snippet="s_cover" data-name="Cover" style="background-image: none; position: relative;" data-oe-shape-data="{'shape':'web_editor/Airy/04_001','flip':[],'showOnMobile':false,'animated':'true'}">
+                            <span class="s_parallax_bg oe_img_bg o_bg_img_center" style="background-image: url('industry_restaurant/static/src/binary/ir_attachment/1379-food.jpg'); background-position: 50% 0px;" data-mimetype="image/webp" data-original-id="1378" data-original-src="industry_restaurant/static/src/binary/ir_attachment/1378-food.jpg" data-mimetype-before-conversion="image/png" data-resize-width="1920"/>
```
Comment (vava-odoo) [inline industry_restaurant/demo/website_view.xml:14]: Do we need the data-original-* values?

File: industry_restaurant/demo/website_view.xml [hunk 2]
```diff
@@ -1,5 +1,82 @@
...
+                            </div>
+                        </section>
+                        <section class="s_image_gallery o_slideshow s_image_gallery_show_indicators s_image_gallery_indicators_rounded o_colored_level pb0 pt40" data-vcss="001" data-columns="3" style="height: 500px; overflow: hidden; background-image: none; position: relative;" data-snippet="s_image_gallery" data-name="Image Gallery" data-oe-shape-data="{'shape':'web_editor/Airy/13_001','flip':[],'showOnMobile':false,'animated':'true'}">
```
Comment (vava-odoo) [inline industry_restaurant/demo/website_view.xml:38]: The carousel doesn't seem to display correctly

**Review by vava-odoo — CHANGES_REQUESTED** [review]
Thanks @dhrs-odoo for this nice work.
Please have a look at the following comment. Please also change the commit title to "[IMP] industry_restaurant: add website homepage".
Cheers

**Comment by dhrs-odoo** [comment]
Hello @vava-odoo 
I have incorporated your suggestions
KIndly please have a look at it once you are free
Thank You :innocent:

**Pull request merged**